### DISCUSSION TITLE
QfieldCloud sync issues

### DISCUSSION THREAD

**Discussion by VilleBio** [opengisch/QField#4015] [Bugs]
We (Enviroplanning_organisation) has recently had problems with syncing data in many of our projects. We've had problems of that edited point-layers simply dissapear after syncronization even though no changes has been made on other connected devices. No changes seem to be registered in the cloud homepage for this one

At the moment we are experiencing problems when working on two mobile android devices and one pc. Some point-objects that are added on mobile devices dissappear after sync. Also some point-objects (from other pointlayers) that are deleted and then pushed into cloud re-appear after sync. We can't figure out the logic of the dissapearances or re-appearances. Many collegues within our organization experiences these problems at the moment. I don't know about any error-messages for these at the moment. Some points sync fine, some don't.

Would be grateful if you could help us fix this, staff at QfieldCloud!

**Comment by GlaDal** [comment]
Are you sure that points didn't exist anymore ? I think that I have the same problem. I don't understand why. For some point, attributes are deleted and the point disappeared because the symbology based on attribute. Can you have a look if it's same problem for you ? My colleague said me this morning that sometimes points disappeared but it's another problem because all attributes are recorded and synchronised. Maybe it's a problem of management of our project using different version of Qgis (3.22 and now 3.28) for synchronising. And one more things, it seems I have the problem with one tablet (the older project created and synchronised) but not with another tablet (where there is the newer installed Qfield and synchronised project). It's the same version of Qgis, Qfield and Qfield (LTR or last one version) and exactly the same tablet. I used a new profile in Qgis for doing the synchronisation.
I hesitate to redo the project from the beginning but it will take me too much time for doing it and I'm afraid to forget something. It's not a good solution. And the problem can appeared in this new version... 
Too much time spent for configuring projects and too much problems not resolved. 
For now, I'm quite bit disappointed...

**Comment by suricactus** [comment]
Hey, @VilleBio and @GlaDal . Please raise the issue using the integrated ticketing system in QFieldCloud so we can investigate your issues. Obviously you are experiencing something that should not be the case. Apologies.

**Reply by VilleBio** [reply]
Where can I find the integrated ticketing system? I have looked in the website (https://app.qfield.cloud/) within the organization page but I cant find it. 

Best regards, Vilhelm

**Reply by VilleBio** [reply]
I've now found it.

**Reply by GlaDal** [reply]
Done !
